docs/concepts/functional-components.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Functional Components
+sidebar_label: Introduction
+description: Introduction to functional components 
+---
+
+:::note
+Functional components are currently unreleased.
+:::
```

Comment:
The previous section had a link to the project board, let's add that here too.
```suggestion
:::warning
Functional components are still under development.
Take a look at the [project board](https://github.com/yewstack/yew/projects/3) for details and consider helping out.
:::
```

Reply:
Why did you place the link further down? The project board isn't specific to hooks or their documentation, it's about yew-functional as a whole.
My suggestion also changes the admonition from "note" to "warning" because we really don't want users to use them yet.

Reply:
It was me trying to add some docs/placeholders for hooks docs too. I've moved the link back above now.